llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -127,6 +127,26 @@ LogicalResult ScatterOp::verify() {
     return op->emitOpError("expected index depth is static");
   }
 
+  auto dimMapTy = getDimensionMap().getType().cast<ShapedType>();
+  if (dimMapTy.getRank() != 1 || dimMapTy.getNumElements() != indexDepth) {
```

Comment:
maybe we want `I64ArrayAttr` for dimMap, so we don't have to check if the rank is 1?

Reply:
Done.